azure-group-deploy.sh

```diff
@@ -106,7 +106,7 @@ azure group create "$resourceGroupName" "$location"
 # Remove line endings from parameter JSON so it can be passed in to the CLI as a single line
 parameterJson=$( echo "$parameterJson" | jq -c '.' )
 
-if [[ -z $validateOnly ]]
+if [[ $validateOnly ]]
```

Comment:
Let's make this consistent with the upload if statement (simpler is fine, just make them the same)